practical/matlab/idr0021_steps.m

```diff
@@ -0,0 +1,203 @@
+% Note: Do not run the whole script. Select the code block
+% of each exercise, right-click and "Evaluate Selection". 
+% Then proceed to the next exercise.
+% The exercises build on top of each other (later exercises
+% cannot be run unless the previous exercises have been 
+% executed successfully). If you get stuck, clear the workspace
+% and try again from the beginning.
+
+% Exercise 1
+% Connect to OMERO and print your group ID.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+client = loadOmero('outreach.openmicroscopy.org');
+session = client.createSession('USER', 'PASSWORD');
+client.enableKeepAlive(60);
+eventContext = session.getAdminService().getEventContext();
+groupId = eventContext.groupId;
+disp(groupId);
+
+
+% Exercise 2
+% List the images of a particular dataset.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+datasetId = MATLAB_DATASET_ID;
+loadedDatasets = getDatasets(session, datasetId, true);
+dataset = loadedDatasets(1);
+datasetName = dataset.getName().getValue();
+disp(datasetName)
+datasetImages = getImages(session, 'dataset', datasetId);
+for i = 1 : length(datasetImages)
+    image = datasetImages(i);
+    fprintf('%s , %i\n', image.getName().getValue(), image.getId().getValue());
+end
+
+
+% Exercise 3
+% Get the target protein name (== name of the relevant channel).
```

Comment:
Mention that you are taking the info about which channel is relevant from the MapAnnoations